### ISSUE TITLE
Uh Oh.

### ISSUE THREAD

**Issue by NetizenAbel** [israel-dryer/Covid-19#2]
I hope this is you and not them ... but I fear I'm wrong.

The data for individual counties near 98321:

1     (6.8 miles)     0        Pierce County, WA
2     (32.0 miles)    0        King County, WA
3     (33.9 miles)    442      Washington

**Comment by israel-dryer** [comment]
Looks right in total. Though, I'm surprised that it's no longer breaking out by county... There are quite a few cases in King County if I remember.  Here's a screenshot of the app by Johns Hopkins. Totals seem to match.

![Screenshot_20200313-005447_Firefox](https://user-images.githubusercontent.com/42584742/76591276-427cd180-64c6-11ea-91a3-53c4f292ca63.jpg)

**Comment by NetizenAbel** [comment]
There were definitely 4 cases in Pierce last night before this started. What happened is that your data is coming from a repo that breaks every couple hours. If this is to work, the parsing will have to deal with constantly changing and shifting political names and the like, as well as data being frequently scrubbed.

So effectively, we still have no data. Nobody but China seems to have any data with any consistency.

**Comment by israel-dryer** [comment]
yeah, not sure why they stopped reporting at lower levels... looks like it's all being rolled to state level now. But, I went ahead and filtered out the records where there is no data to at least make it somewhat useful.

![image](https://user-images.githubusercontent.com/42584742/76658078-abf1f400-6549-11ea-96eb-eb31179bf02f.png)

But, at this point, there are a lot of good sources out there, including this one I found earlier today.
https://public.tableau.com/profile/covid.19.data.resource.hub#!/vizhome/COVID-19Cases_15840488375320/COVID-19Cases

**Comment by NetizenAbel** [comment]
You're doing helpful work, thank you.
